pkg/compactor/compactor_test.go

```diff
@@ -1644,6 +1644,9 @@ func prepareConfig() Config {
 	// Set lower timeout for waiting on compactor to become ACTIVE in the ring for unit tests
 	compactorCfg.ShardingRing.WaitActiveInstanceTimeout = 5 * time.Second
 
+	// Set CachingBucketEnabled to true
+	compactorCfg.CachingBucketEnabled = true
```

Comment:
Seems like it doesn't make any difference because we didn't set any cache backend. And it only supports memcached and redis, not inmemory cache, which makes it hard to run in unit test.

Let's remove this for now? We need to have proper integration tests for compactor in the future